src/networks.ts

```diff
@@ -0,0 +1,15 @@
+import { ETH_RPC, ETH_RPC_MAINNET } from '@big-whale-labs/constants'
+import Network from 'models/Network'
+import createProvider from 'helpers/providers/createProvider'
+import env from 'helpers/env'
+
+export default {
+  [Network.Mainnet]: createProvider(Network.Mainnet, {
+    default: ETH_RPC_MAINNET,
+    heavy: env.VITE_ETH_RPC_MAINNET,
+  }),
+  [Network.Goerli]: createProvider(Network.Goerli, {
+    default: ETH_RPC,
```

Comment:
Same question here

Reply:
it's better to leave the constants for now